include/directx/d3d12shader.h

```diff
@@ -208,54 +260,95 @@ typedef struct _D3D12_SHADER_INPUT_BIND_DESC
 #define D3D_SHADER_REQUIRES_SAMPLE_CMP_GRADIENT_OR_BIAS                                     0x80000000
 #define D3D_SHADER_REQUIRES_EXTENDED_COMMAND_INFO                                           0x100000000ull
 
-
 typedef struct _D3D12_LIBRARY_DESC
 {
     LPCSTR    Creator;           // The name of the originator of the library.
     UINT      Flags;             // Compilation flags.
     UINT      FunctionCount;     // Number of functions exported from the library.
 } D3D12_LIBRARY_DESC;
 
+typedef enum D3D12_NODE_OVERRIDES_TYPE {

```

Comment:
`D3D12_NODE_OVERRIDES_TYPE` is a D3D12 enum for specifying which parameter overrides are specified in a union passed to the API when constructing a graph.  It may look like an enum that defines launch type, but that's not its purpose.

For example, `D3D12_NODE_OVERRIDES_TYPE_COMMON_COMPUTE` has no corresponding launch type, instead indicating that overrides common to any launch type are being specified.  Imagine a new hypothetical launch type was added - just for fun let's call it 'mesh' and assign it a value of 4.  That would collide with the COMMON_COMPUTE enum value.

`D3D12_NODE_SHADER_DESC` should use an enum specifically defining launch type, like `D3D12_NODE_LAUNCH_TYPE`, rather than reuse the `D3D12_NODE_OVERRIDES_TYPE` enum.

Reply:
Done. The only thing I'm not 100% sure about is the DXC backend. At the time of CFunctionReflection being available, is D3D12_NODE_OVERRIDES_TYPE_COMMON_COMPUTE  already resolved or do I need to explicitly catch it and turn it into a different D3D12_NODE_LAUNCH_TYPE?

Reply:
Okay good, should be good to go then :)